package.json

```diff
@@ -68,7 +68,7 @@
     "reset": "pnpm clean:all; rm pnpm-lock.yaml; pnpm symlink:root; pnpm install;",
     "run:prettier": "prettier --ignore-path .eslintignore \"**/*.{js,jsx,ts,tsx,md,mdx,json,html,css,yml,yaml,graphql}\"",
     "size": "size-limit",
-    "storybook": "manypkg run support/storybook storybook",
+    "storybook": "cross-env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\"}' manypkg run support/storybook storybook",
```

Comment:
I wasn't aware of this. Has it changed recently?

Reply:
TBH: I really don't know. Not providing this produces errors from babel, so it could also be that a babel default has changed ... or maybe something in react-scripts/webpack? This was a quick-and-easy way to fix things up, and is nice and localized to not disturb anything else. :)